### ISSUE TITLE
make "no-multi-spaces" work in JSDoc / comments

### ISSUE THREAD

**Issue by mobidev111** [eslint/eslint#8917]
JSDoc comment formatting can get messy with multiple spaces. the JSDoc is valid, though.

**Tell us about your environment**

* **ESLint Version:**
4.2.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**What did you do? Please include the actual source code causing the issue.**

```js
/*eslint no-multi-spaces: "error"*/
/**
 * @param  {Object1} NamedObject2
 * @param {Object2} NamedObject2
 */
```
(two spaces '  ' between @param and {Object1})

**What did you expect to happen?**
rule "no-multi-spaces" should throw error for '@param  {Object1}'

**What actually happened? Please include the actual, raw output from ESLint.**
rule "no-multi-spaces" does not throw error for '@param  {Object1}'

**Comment by not-an-aardvark** [comment]
Thanks for the proposal. Would you expect an error to be thrown for something like this? I'm worried that this wouldn't be what people would generally want:

```js
/**
 * @param {foo} bar
 * @example
 *
 * function foo() {
 *   bar(); // <-- three consecutive spaces on this line
 * }
 */
```

**Comment by mobidev111** [comment]
Indeed, in this case throwing an error wouldn't help.

Probably could be addressed by focusing on selected use cases - instead of tackling any consecutive space in a comment. For each of these use cases an option could be added to the "no-multi-spaces" rule.

for the JSDoc use case I'm mentioning the rule could be: remove consecutive spaces from JSDoc tags
* example: '@param' tag http://usejsdoc.org/tags-param.html
* probably similar for other JSDoc tags: http://usejsdoc.org
--> could be as well an extension to the "valid-jsdoc" rule

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get consensus from the team, so I'm closing it. We define [consensus](http://eslint.org/docs/maintainer-guide/issues.html#consensus) as having three :+1:s from team members, as well as a team member willing to champion the proposal. This is a high bar by design -- we can't realistically accept and maintain every feature request in the long term, so we only accept feature requests which are useful enough that there is consensus among the team that they're worth adding.

Since ESLint is pluggable and can load custom rules at runtime, the lack of consensus among the ESLint team doesn't need to be a blocker for you using this in your project, if you'd find it useful. It just means that you would need to [implement the rule](http://eslint.org/docs/developer-guide/working-with-rules) yourself, rather than using a bundled rule that is packaged with ESLint.

**Comment by platinumazure** [comment]
I'm not sure about enhancing `no-multi-spaces` for this use case, but I think it could make sense to create another rule around validating spacing in JSDoc comments in specific areas. Perhaps someone could create a custom rule around this? Or, if someone wants to discuss this possibility I have raised as a potential core ESLint rule, please feel free to create a new issue.
